From 1b0c2d842ce92ea350a681a8f3ed6733e7094318 Mon Sep 17 00:00:00 2001
From: Kohei Yoshida <kohei.yoshida@gmail.com>
Date: Wed, 15 Aug 2012 14:06:06 -0400
Subject: [PATCH 1/3] Do not check the stored DIFAT sector count.

Refer to this thread for details.
http://lists.freedesktop.org/archives/libreoffice/2012-August/036852.html

Change-Id: Id2115732a1e1bcd78c13445f5aaefe1ec7cfa0c2
(cherry picked from commit 50d89530eee57afb890e2e65aac8a5b0f93a74a3)

Signed-off-by: Ivan Timofeev <timofeev.i.s@gmail.com>
(cherry picked from commit 1053dada25491ba5609ae47bef8766e379e96fd8)
---
 sot/source/sdstor/stgelem.cxx | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sot/source/sdstor/stgelem.cxx b/sot/source/sdstor/stgelem.cxx
index 1af72f9..90f1e2d 100644
--- a/sot/source/sdstor/stgelem.cxx
+++ b/sot/source/sdstor/stgelem.cxx
@@ -203,7 +203,7 @@ sal_Bool StgHeader::Check()
             && nTOCstrm >= 0
             && nThreshold > 0
             && ( nDataFAT == -2 || ( nDataFAT >= 0 && nDataFATSize > 0 ) )
-            && ( nMasterChain == -2 || ( nMasterChain >=0 && nMaster > 109 ) )
+            && ( nMasterChain == -2 || nMasterChain >=0 )
             && nMaster >= 0;
 }
 
-- 
1.7.11.7

